spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -95,10 +95,33 @@
     and_I_see_a_step_deleted_success_notice
   end
 
+  scenario "User deletes a draft step by step guide without change notes" do
+    given_there_is_a_draft_step_by_step_page
+    and_I_visit_the_publish_or_delete_page
+    and_I_delete_the_draft
+    then_I_see_a_step_by_step_deleted_success_notice
+  end
+
+  scenario "User deletes a draft step by step guide with change notes" do
+    given_there_is_a_draft_step_by_step_page
+    and_it_has_change_notes
+    and_I_visit_the_publish_or_delete_page
+    and_I_delete_the_draft
+    then_I_see_a_step_by_step_deleted_success_notice
+  end
+
   def given_there_is_a_published_step_by_step_page
     @step_by_step_page = create(:published_step_by_step_page)
   end
 
+  def given_there_is_a_draft_step_by_step_page
```

Comment:
This does creates the same object as `given_there_is_a_step_by_step_page` so you just reuse that.